src/Ng/Scripts/Catalog2LuceneV3-SouthAsia.cmd

```diff
@@ -0,0 +1,25 @@
+@echo OFF
+
+cd Ng
+
+:Top
+echo "Starting job - #{Jobs.catalog2lucenev3reg2southasia.Title}"
```

Comment:
`southasia` seems a bit odd. Perhaps `southeastasia`? Let's adhere to precedent

Reply:
Fixed